File: customer/src/dtos/externalEvents/customer-was-created.ts
export interface CustomerWasCreated {
    id?: string;
    name?: string;
    motherName?: string;
    birthDate?: google.protobuf.Timestamp;
    created?: google.protobuf.Timestamp;
    status?: CustomerStatusType;
}
export enum CustomerStatusType {
    None = 0,
    Simple = 1,
    Partial = 2,
    Complete = 3,
}
export namespace google {
    export namespace protobuf {
        export interface Timestamp {
            seconds?: number;
            nanos?: number;
        }
    }
}

File: antifraud/Antifraud/ExternalContracts/CustomerWasCreated.cs
using System;

namespace Antifraud.ExternalContracts
{
    public class CustomerWasCreated
    {
        public string Id { get; set; }
        public string DocumentNumber { get; set; }
        public string Name { get; set; }
        public string MotherName { get; set; }
        public CustomerStatusType Status { get; set; }
    }
}